ge/org.osate.ge/src/org/osate/ge/BusinessObjectSelection.java

```diff
@@ -40,4 +41,11 @@
 	 */
 	<T extends EObject> void modify(Class<T> c, Consumer<T> modifier);
 
+	/**
+	 * Adds a modification for each business object to the operation being constructed using opBuilder
+	 * Throws an exception if any of the objects in the set are not of the specified type.
+	 */
+	<T extends EObject, O> void modifyWithOperation(final OperationBuilder<O> opBuilder, Class<T> c,
```

Comment:
Either remove final from opBuilder, or add final to other parameters.  Removing final would be consistent with the other methods.